packages/glossary-plugin/src/plugin.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import GlossaryPopup from "./components/glossary-popup";
+
+interface IExternalScriptContext {
+  div: any;
+}
+
+export class GlossaryPlugin {
+  constructor(authoredState: any, context: IExternalScriptContext) {
```

Comment:
Just noting that we had talked about eventually moving authoredState into context in the final API.  Just making a note of that here for later.

Reply:
Sure, I'm just using current ExternalScript API so I can test and demo things. I'll obviously switch to the new one when enough functionality is ready.